blockonomics.php

```diff
@@ -200,9 +202,27 @@ public function hookPaymentOptions($params)
     public function getPaymentOption()
     {
         $offlineOption = new PaymentOption();
+        $active_cryptos = $this->getActiveCurrencies();
+        $logoHeight = Configuration::get('BLOCKONOMICS_LOGO_HEIGHT');
+
+        $this->context->smarty->assign('blockonomicsLogoHeight', $logoHeight);
+        
+        $cryptos = array();
+        $logo_icons = array();
+        foreach ($active_cryptos as $crypto) {
+            array_push($cryptos, $crypto['name']);
+            array_push($logo_icons, 
+                _MODULE_DIR_.'blockonomics/views/img/'.$crypto['code'].'-icon.png'
+            );
+        }
+
+        $this->context->smarty->assign('blockonomicsEnabledLogos', $logo_icons);
+        
         $offlineOption
-            ->setCallToActionText($this->l('Pay by bitcoin'))
-            ->setLogo(_MODULE_DIR_.'blockonomics/views/img/bitcoin-icon.png')
+            ->setCallToActionText($this->l('Pay by ' . join(' or ', $cryptos)))
+            ->setAdditionalInformation(
+                $this->context->smarty->fetch('module:blockonomics/views/templates/front/logo_height.tpl')
```

Comment:
The file location can be moved to the more appropriate module:blockonomics/views/templates/**hook**/logo_height.tpl, as this is used in hookPaymentOptions.

Reply:
Okay